src/components/bulkConcepts/component/ActionButtons.jsx

```diff
@@ -40,14 +41,21 @@ export class ActionButtons extends Component {
   addConcept = (conceptUrl, name) => {
     const { params } = this.props;
     const data = { data: { expressions: [conceptUrl] } };
-    this.props.addConcept(params, data, name);
+    if (this.props.mappings && this.props.mappings.length > 0) {
+      this.props.addConcept(params, data, name);
+      this.props.mappings.map((concept) => {
```

Comment:
Is there a way of adding multiple references to concepts in one go? This seems like it will not scale well if we keep making an api call for each mapping.

Reply:
You are right Lincoln, multiple API calls could slow down the application. I will figure out a work around